### ISSUE TITLE
[Feature request]: Consistent shortflags

### ISSUE THREAD

**Issue by meunierd** [flatpak/flatpak#4893]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a feature request that matches the one I want to file, without success.

### Suggestion

Wherever there is no collision, it would be great to have shortflags (`-`) for all of the `--` style arguments.

ie:

```
$ flatpak search --help
Usage:
  flatpak search [OPTION…] TEXT - Search remote apps/runtimes for text

Help Options:
  -h, --help              Show help options

Application Options:
  -u, --user                  Work on the user installation
  -s, --system                Work on the system-wide installation (default)
  -i, --installation=NAME     Work on a non-default system-wide installation
  -a, --arch=ARCH             Arch to search for
  -c, --columns=FIELD,…       What information to show
  -v, --verbose               Show debug information, -vv for more detail
  -o, --ostree-verbose        Show OSTree debug information
```
I think it's consistent, and it's less typing for when you're actually invoking these from the terminal versus writing it out "longhand" for documentation purposes in say a script to install all of my flatpaks onto a fresh install.

Using the search feature in particular, to actually install something I've searched for, I ensure that the application ID is not truncated by an ellipsis by always specifying `--columns`.

**Comment by chrisawi** [comment]
I don't think we want to get carried away with assigning shortflags. There are only so many letters available, and excessive usage of them will hurt readability, especially if they have different meanings between commands.

`-u` and `-v` already exist. The other one that might be useful is `-c`, except that you still have to specify the column names, so it doesn't save much typing.

Looking broadly at other commands, I think `pin` and `mask` could use `-r` for `--remove`. Also, `info` is missing `-u` for some reason.

**Comment by smcv** [comment]
> I don't think we want to get carried away with assigning shortflags. There are only so many letters available, and excessive usage of them will hurt readability, especially if they have different meanings between commands.

Yes, this.

In particular, I think a situation like this one would be really bad and confusing:

* `flatpak foo` has a `-a` option which is short for `--all`
* `flatpak bar` has a `-a` option which is short for `--automatic`
* now we develop new functionality for `flatpak bar` which gives it an `--all` option
* ... but we can't make `flatpak bar -a` an alias for `flatpak bar --all`, because `-a` already means `--automatic`
* so the meaning of `flatpak foo -a` and `flatpak bar -a` will be inconsistent, and will remain inconsistent forever

I think we should generally only be assigning short (single-letter) option names for things that are genuinely so common to type that a long option is a burden, preferring to use short option names that are available in many other CLI tools to make them more obvious/more discoverable, like `--verbose`/`-v`.
